Fix the ./configure --enable-static-xv parameter, that was broken on 1.1.1
release.
Index: xine-lib-1.1.1/configure.ac
===================================================================
--- xine-lib-1.1.1.orig/configure.ac
+++ xine-lib-1.1.1/configure.ac
@@ -559,10 +559,14 @@ dnl Only dynamic linking is possible whe
 AC_ARG_WITH(xv-path, AC_HELP_STRING([--with-xv-path=path], [where libXv is installed]),
             xv_path="$withval",)
 
-AC_ARG_ENABLE(shared-xv,
-        AC_HELP_STRING([--enable-static-xv],[Enable this to force linking against libXv.a]),
-        xv_prefer_shared="no",
-        xv_prefer_shared="yes")
+AC_ARG_ENABLE([static-xv],
+        AC_HELP_STRING([--enable-static-xv],[Enable this to force linking against libXv.a]))
+
+if test "x$enable_static_xv" = "xyes"; then
+  xv_prefer_shared="no"
+else
+  xv_prefer_shared="yes"
+fi
 
 if test x"$no_x" != "xyes"; then
   AC_FIND_LIBXV
